CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
-# Release Notes
+# Cloud Code for Visual Studio Code Release Notes
 
 This page documents production updates to Cloud Code for Visual Studio Code. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
-## Version 1.7.2 Release Notes
+## Version 1.7.2 Release Notes: January 15, 2021
 
-### Fixes 
-* [Issue #362](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/362) Fixes an issue where environmental path delimiter can invalidate the path on Windows.
-* Fixes an issue where Cloud SDK sometimes gets corrupted on Windows since there is reentry while installing dependencies.
+### Bug Fixes 
+* Fix an issue where dependencies path gets invalidated on Windows. [#362](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/362) 
+* Fix an issue where Cloud SDK in the path gets corrupted on Windows since there is reentry while installing dependencies. [#347](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/347)
 
-## Version 1.7.1 Release Notes
+## Version 1.7.1 Release Notes: December 09, 2020
```

Comment:
Wasn't this multi-lined in the proposal (header followed by the date)?

Reply:
That doc format is WIP and not yet finalized, currently following the cpp extension and that seemed more readable.

Reply:
Chairy updated the recommendation in the doc to "version (date)" format, the update is to reflect to that format.